deploy/olm-catalog/kogito-operator/0.13.0/app.kiegroup.org_kogitotrusties_crd.yaml

```diff
@@ -0,0 +1,370 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Small heads up: we should rebase this after 0.13 release and do not add it to this version.

Reply:
:+1: